### ISSUE TITLE
Can't Find File in Swift Package, Vapor Project

### ISSUE THREAD

**Issue by mikaelacaron** [DanielEdrisian/AlexSideBar-Public#24]
**Describe the bug**
Alex wasn't able to find a file in my Vapor project (which Vapor is a Swift package)

using `@` and typing the filename, or explicitly going to + then Files and searching, it didn't find it

and looking at the file path it shows on the right I have two projects that both start in the same directory. One is the iOS app and then one is the vapor project, and actually the files it found were all iOS project files.
* FruitfulApp
   * Fruitful (iOS app)
   * FruitCore (Vapor backend)

I did actually have both projects open, but when I closed the iOS app, it still had the same issue

**Screenshots/Screen recordings**
![Screenshot 2024-11-24 at 2 08 48 AM](https://github.com/user-attachments/assets/71f37cde-44b9-46c0-bc02-6c7953d1a555)

**What version are you on currently?**
 - Version 1.2 (66)

**Comment by DanielEdrisian** [comment]
@mikaelacaron Can you try re-indexing the whole project (click on "FruitCore" name on the top of Alex, then click delete, and then load). The wait and search for the file.

**Comment by DanielEdrisian** [comment]
It might be the case that it's not indexing local Swift packages. I'll check.

**Comment by mikaelacaron** [comment]
yeah it looks like it's not in the right directory, indexing Swift packages, it's in `FruitfulApp`, not in the `FruitCore` directory (which has the Vapor project), like it should be, and I searched again, and it didn't find the file

![Screenshot 2024-11-24 at 2 45 40 PM](https://github.com/user-attachments/assets/c2b66013-2db2-4940-b9ac-15bb201edf2f)
